neon_mq_connector/connector.py

```diff
@@ -174,6 +212,16 @@ def emit_mq_message(cls, connection: pika.BlockingConnection, queue: str, reques
         else:
             raise ValueError(f'Invalid request data provided: {request_data}')
 
+    @classmethod
+    def publish_message(cls,
+                        connection: pika.BlockingConnection,
+                        request_data: dict,
+                        exchange: Optional[str] = '',
+                        expiration: int = 1000) -> str:
+        """Convenience method to implement publish event on fanout exchange, wrapper for emit_mq_message()"""
```

Comment:
Consider copying docstring from `emit_mq_message` for param and return descriptions